src/benchmark/Akka.Benchmarks/Actor/ActorPathBenchmarks.cs

```diff
@@ -20,18 +21,25 @@ public class ActorPathBenchmarks
         private Address _sysAdr = new Address("akka.tcp", "system", "127.0.0.1", 1337);
         private Address _otherAdr = new Address("akka.tcp", "system", "127.0.0.1", 1338);
 
+        private string _actorPathStr;
+        
+        [Params(1, 100000, int.MaxValue)]
+        public int Uid { get; set; }
```

Comment:
Parameterize the UID using increasing lengths - `int.MaxValue` is the upper limit of what the UID function will produce in v1.4: https://github.com/akkadotnet/akka.net/blob/680e10129b8a8ec83dd7115d80e1461783dc9ad7/src/core/Akka/Actor/ActorCell.cs#L321-L329